mapie/calibrators/ccp/utils.py

```diff
@@ -77,27 +67,20 @@ def compile_functions_warnings_errors(
     functions: List[Callable]
 ) -> None:
     """
-    Raise warnings and errors if the elements in ``functions`` have
+    Raise errors if the elements in ``functions`` have
```

Comment:
Does this mean this is a `check_functions`? If so, for clarity with the rest of the library I would call it that way.

Reply:
You are write, I renamed it